public/css/1440p.css

```diff
@@ -0,0 +1,124 @@
+@font-face {
```

Comment:
suggestion(blocking): There is overlap in these css files. `body`, `@font-face` and some others can be moved to a base CSS file that gets loaded regardless of the resolution. This will clean up the files, reduce the duplication while making it easier to distinguish what is actually different between the resolutions.